cmake/QtProtobufGen.cmake

```diff
@@ -143,6 +143,7 @@ function(qtprotobuf_generate)
             OUTPUT ${GENERATED_SOURCES_FULL} ${GENERATED_HEADERS_FULL}
             COMMAND ${PROTOC_COMMAND}
                 --plugin=protoc-gen-qtprotobufgen=${QT_PROTOBUF_EXECUTABLE}
+                --experimental_allow_proto3_optional
```

Comment:
I suggest adding a parameter ALLOW_PROTO3_OPTIONAL to a qtprotobuf_generate, to enable '--experimental_allow_proto3_optional' option.